great_expectations/rule_based_profiler/domain_builder/categorical_column_domain_builder.py

```diff
@@ -45,6 +45,8 @@ def __init__(
         ] = None,
         include_column_names: Optional[Union[str, Optional[List[str]]]] = None,
         exclude_column_names: Optional[Union[str, Optional[List[str]]]] = None,
+        semantic_type_filter_module_name: str = "great_expectations.rule_based_profiler.helpers.simple_semantic_type_filter",
+        semantic_type_filter_class_name: str = "SimpleSemanticTypeFilter",
```

Comment:
any chance we can use the actual type of the class (as opposed to the string)? perhaps not in this context but I'd love to use types instead of strings where possible because they are susceptible to typos and other misc errors.

Reply:
@cdkini We cannot -- for the reasons of `Marshmallow` `Schema` compatibility.